go.mod

```diff
@@ -3,3 +3,9 @@ module github.com/notaryproject/notation-core-go
 go 1.18
 
 require github.com/golang-jwt/jwt/v4 v4.4.2
+
+require (
+	github.com/fxamacker/cbor/v2 v2.4.0 // indirect
+	github.com/veraison/go-cose v1.0.0-rc.1
+	github.com/x448/float16 v0.8.4 // indirect
+)
```

Comment:
We should have a `require` block of direct dependencies and a `require` block of indirect dependencies since go `1.17`.